index.html

```diff
@@ -2272,12 +2272,29 @@ <h2 style="color: var(--appimage);">Simple and Compund Interest</h2>
                 <table class="table">
                     <tr>
                         <td>
-                            <h5>App is Developed By <span class="badge badge-light" style="cursor:pointer;"
-                                    onclick="window.open('https://sairish2001.github.io')">Sairish</span>
+                            <h5>MakesMathEasy Solves various Math Problems along with Steps. We are providing Complex
+                                Calculations within a few milliseconds along with speed. We work together to
+                                design, create and produce work that we are proud of. We believe that <span
```

Comment:
This about info is great! Just a few changes I'd like to suggest:

> MakesMathEasy solves various math problems along with steps. We provide speedy calculations, within a few. We work together, design, create and produce work that we are proud of. We believe that

The "MATH IS FUN!" part is great, no need to change that :)

Reply:
Great job!

Reply:
TYSM😄